apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx

```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
+| **Last commit**       | Displays the most recent commit to your store's repository, including the date, month, year, and the GitHub user who made the commit. |
+| **Status**            | Displays the status of the deployment.                                                                                                |
+
+The **Status** field can have one of the following values: 
+- `Success`: The deployment was successful. If you want to check the commit associated with it, go to the **Deploys** tab.
+
+- `Error`: There was an error in the pipeline that caused the deployment to fail, but the build can still be successful. 
+To understand what happened, go to the **Deploys** tab, check the last commit, and open the repository for further investigation.
+
+--- 
+
+## Deploys
+
+The **Deploys** view provides a summary of each deployment made in your store's repository.
+
+![deploy-view](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-view-deploys___e5f04f3e5613d52e4b2864ef48134608.png)
+
+| Field                         | Description                                                                                                                  |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------- |
+| **Preview URL**               | Displays a store preview for a specific commit.                                                                                          |
+| **Commit**                    | Displays the commit hash associated with a particular commit.                                                                                     |
+| **Timestamp**                 | Displays the date and time of the commit in the format `dd/mm/yyyy`.                                                     |
+| **Elapsed time**              | Displays the time it took to complete the commit process.   |
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Failed`. For more information on the `Failed` status, go to the repository and check what failed in the commit.                   |
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are <li>`Success` : All checks have passed.</li> <li>`Skipped`:  In the CI process, specific checks were not performed for certain commits This is often done when the commit being tested was never the HEAD commit of the main branch, saving time and resources without compromising code quality.</li> <li>`Failed` : For more information on the `Failed` status, go to the repository and check what failed in the commit.</li>  *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass.                     |
```

Comment:
```suggestion
| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are <ul><li>`Success` : All checks have passed.</li> <li>`Skipped`:  In the CI process, specific checks were not performed for certain commits. This often occurs when the commit being tested was never the HEAD commit of the main branch, saving time and resources without compromising code quality.</li> <li>`Failed` : For more information on the `Failed` status, go to the repository and check what failed in the commit.</li></ul>  *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the Lighthouse checks to pass.                     |
```

Reply:
"For more information on the `Failed` status, go to the repository and check what failed in the commit" - Teria que ver o que falhou na audicão do Lighthouse, não? Depois a pessoa vai investigar e corrigir o código dela